src/imgproc/imgproc.cpp

```diff
@@ -10,6 +10,43 @@
 #include <memory>
 #include <vector>
 
+Vec6f convertToVec6f(const cv::Vec6f& cvVec) {
+    Vec6f vec;
+    vec.val1 = cvVec[0];
+    vec.val2 = cvVec[1];
+    vec.val3 = cvVec[2];
+    vec.val4 = cvVec[3];
+    vec.val5 = cvVec[4];
+    vec.val6 = cvVec[5];
+    return vec;
+}
+
+CvStatus GetTriangles(VecPoint points, Vec6f **rval, int *size)
+{
+    BEGIN_WRAP
+    // get convex hull from potins
+    // cv::convexHull(*points.ptr, *hull.ptr, clockwise, returnPoints);
+    cv::Mat convexHull;
+    cv::convexHull(*points.ptr, convexHull);
+    //    cv::Rect r = cv::boundingRect(*pts.ptr);
+    cv::Rect r = cv::boundingRect(convexHull);
+    cv::Subdiv2D subdiv = cv::Subdiv2D(r);
+    for (int i = 0; i < points.ptr->size(); i++) {
+        //        points.ptr->at(i);
+        cv::Point2f pt = cv::Point2f(points.ptr->at(i).x, points.ptr->at(i).y);
+        subdiv.insert(pt);
+    }
+    std::vector<cv::Vec6f> triangleVec;
+    subdiv.getTriangleList(triangleVec);
+
+    *size = triangleVec.size();
+    // rval에 triangleVec 할당
+    *rval = new Vec6f[triangleVec.size()];
+    for (size_t i = 0; i < triangleVec.size(); i++) {
+        (*rval)[i] = convertToVec6f(triangleVec[i]);
+    }
+    END_WRAP
+}
```

Comment:
I understand that you may just need `getTriangles` function from [Subdiv2D](https://docs.opencv.org/4.x/df/dbf/classcv_1_1Subdiv2D.html#aed58be264a17cdbe712b6a35036d13cb) class so only `getTriangles` was added, but `getTriangles` is not static so we have to create a new `Subdiv2D` instance every time the function is called, so wrapping Subdiv2D class is a better solution, as shown in [objdetect](https://github.com/rainyl/opencv_dart/blob/8deb960fcea74de4ba393c52ad0f40cd4a3a4168/src/objdetect/objdetect.h#L21C1-L42C70)

Reply:
So would you please make more robust and universal by wrapping the class `Subdiv2D`? 

Similar to:
```c++
#ifdef __cplusplus
CVD_TYPEDEF(cv::CascadeClassifier, CascadeClassifier)
#else
CVD_TYPEDEF(void, CascadeClassifier)
#endif

CVD_TYPEDEF_PTR(CascadeClassifier)

// CascadeClassifier
CvStatus CascadeClassifier_New(CascadeClassifier *rval);
void     CascadeClassifier_Close(CascadeClassifier *cs);
CvStatus CascadeClassifier_Load(CascadeClassifier cs, const char *name, int *rval);
CvStatus CascadeClassifier_DetectMultiScale(CascadeClassifier cs, Mat img, VecRect *rval);
CvStatus CascadeClassifier_DetectMultiScaleWithParams(CascadeClassifier cs, Mat img, double scale,
                                                      int minNeighbors, int flags, Size minSize, Size maxSize,
                                                      VecRect *rval);
```